### ISSUE TITLE
Move aria-label from div to button

### ISSUE THREAD

**Issue by erinchambers** [CDRH/orchid#324]
Flagged by axe DevTools:

> **Elements must only use permiddted ARIA attributes**: `aria-label` attribute cannot be used on a `div` with no valid `role` attribute.

# Suggested fix

The `aria-label` in question should be on the link with `role="button", not the parent div.

In this code, for example:

```
<div class="input-group-btn" title="Clear Places Filter" aria-label="Clear Places Filter">
  <a class="btn btn-default btn-sm" rel="nofollow" role="button" href="/ardhi/search">
    <span class="glyphicon glyphicon-remove" aria-hidden="true"></span>
  </a>          
</div>
```

... `aria-label="Clear Places Filter" should be on the `<a role="button">`, like so:

```
<div class="input-group-btn" title="Clear Places Filter">
  <a class="btn btn-default btn-sm" rel="nofollow" role="button" aria-label="Clear Places Filter" ref="/ardhi/search">
    <span class="glyphicon glyphicon-remove" aria-hidden="true"></span>
  </a>          
</div>
```

This solves the error in #325 as well.
